app/src/main/res/values/dimens.xml

```diff
@@ -206,9 +206,9 @@
     <dimen name="account_settings_device_name_min_height">48dp</dimen>
 
     <!-- Top sites  -->
-    <dimen name="top_sites_favicon_size">40dp</dimen>
-    <dimen name="top_sites_favicon_padding">4dp</dimen>
-    <dimen name="top_sites_item_size">64dp</dimen>
+    <dimen name="top_sites_favicon_size">60dp</dimen>
+    <dimen name="top_sites_favicon_padding">12dp</dimen>
+    <dimen name="top_sites_item_size">84dp</dimen>
```

Comment:
This is the height. We will need to adjust this in a separate issue when we make the label be more than 1 line.

<img width="87" alt="Screen Shot 2021-04-08 at 4 22 40 PM" src="https://user-images.githubusercontent.com/1190888/114091436-ae2a9d00-9886-11eb-9e8b-3b066cfff969.png">

Reply:
Thank you! This was really helpful,appreciated.